pygeoapi/api/__init__.py

```diff
@@ -628,6 +628,9 @@ def get_format_exception(self, request) -> Tuple[dict, int, str]:
             HTTPStatus.BAD_REQUEST, headers,
             request.format, 'InvalidParameterValue', msg)
 
+    def get_base_url(self) -> str:
```

Comment:
We already have `pygeoapi.util:get_base_url` (which is used to set `self.base_url`, suggest to remove this function.